src/ids.rs

```diff
@@ -555,7 +555,7 @@ def_id!(
 def_id!(PersonId, "person_");
 def_id!(PlanId: String); // N.B. A plan id can be user-provided so can be any arbitrary string
 def_id!(PlatformTaxFeeId, "ptf");
-def_id!(PriceId, "price_" | "plan_"); // see #470
+def_id!(PriceId, "price_" | ""); // see #470
```

Comment:
Since this will basically behave like the `PlanId` and `ProductId`, that being: "it can be any arbitrary string", I would say that maybe it should be the same as them?

So something like:
```suggestion
def_id!(PriceId, String);
```

Also not sure if the link to the issue #470 here will make sense anymore. But could be nice to add a comment explaining about why the `PriceId` can be any String? Or a link to your original issue. Wdyt?

Reply:
Yes, please change this to a string and add a comment. Thanks!